### PR TITLE
Evaluate website against accessibility.

### PR DESCRIPTION
**Pull request by valt19**
# On this milestone, the following have been implemented
- Page titles: change to be more descriptive.
- Image text alternatives: updated the img src links with alt text.
- Text headings:  update headings to a meaningful hierarchy.
- Color contrast: no issues found.
- Resize: no issues found.
- Interaction: update icons with transitions and transform in CSS.  
- Moving content: add the effect on buttons in CSS.
- Multimedia: no issues found.
- The Basic structure of the page:   no issues found.

## Below are the links to the changes made.
https://github.com/valt19/porfolio-/pull/9#discussion_r1121312599
https://github.com/valt19/porfolio-/pull/9#discussion_r1121313329
https://github.com/valt19/porfolio-/pull/9#discussion_r1121314030
https://github.com/valt19/porfolio-/pull/9#discussion_r1121320013
https://github.com/valt19/porfolio-/pull/9#discussion_r1121320255

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @valt19, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that hasn't any commits of your coding partner kindly inform him/her to commit in your repo because it is a pair-programming project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio</title>
+  <title> Web developers Portfolio  </title>
```
Comment (mercymugambi) [inline index.html:7]: Update the title to be more descriptive

File: index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
       
       <a href="welcome" class="head">Welcome</a>
       <nav class="menu-bar">
-        <img src="images/Normal Button/Enabled.png" alt="">
+        <img src="images/Normal Button/Enabled.png" alt=" humburger-button">
```
Comment (mercymugambi) [inline index.html:18]: Add alt text to images url

File: index.html [hunk 3]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
     </div>
       <ul class="iconns"> 
         <li>
-          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt=""></a>  
+          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt="twitter"></a>
```
Comment (mercymugambi) [inline index.html:37]: Add alt text to image url

File: index.html [hunk 4]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
     </div>
       <ul class="iconns"> 
         <li>
-          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt=""></a>  
+          <a href="" class="icon"> <img src="images/icons/twitter.svg" alt="twitter"></a>  
         </li>  
          <li>
-          <a href="" class="icon"> <img src="images/icons/i.png" alt="">
-            <img src="images/icons/n.png" alt="">
+          <a href="" class="icon"> <img src="images/icons/i.png" alt="linked in ">
```
Comment (mercymugambi) [inline index.html:40]: Add alt text to image url

File: index.html [hunk 5]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
+            <img src="images/icons/n.png" alt=" linked in">
            </a> 
          </li>
         <li>
-          <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=""></a>
+          <a href="" class="icon">  <img  class="medium" src="images/icons/medium@2x.png" alt=" medium"></a>
```
Comment (mercymugambi) [inline index.html:45]: Add alt text to image url

File: index.html [hunk 6]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
         </li>
         <li>
-          <a href="" class="icon"><img src="images/icons/github.svg" alt=""></a>
+          <a href="" class="icon"><img src="images/icons/github.svg" alt=" github"></a>
```
Comment (mercymugambi) [inline index.html:48]: Add alt text to image url

File: index.html [hunk 7]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
         </li>
         <li>
-          <a href="" class="icon"><img src="images/icons/angellist.svg" alt=""></a>
+          <a href="" class="icon"><img src="images/icons/angellist.svg" alt=" angellist"></a>
```
Comment (mercymugambi) [inline index.html:51]: Add alt text to image url

File: index.html [hunk 8]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
         </li>
          
       </ul>
       <div class="scroll">
-        <img src="images/Normal Button/Disabled.png" alt="">
+        <img src="images/Normal Button/Disabled.png" alt=" scroll-button">
```
Comment (mercymugambi) [inline index.html:56]: Add alt text to image url

File: index.html [hunk 9]
```diff
@@ -34,31 +34,31 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
       </div>
   </section>
   <section class="all-works">
   
-      <div class="recent-works"><h3 class="hey-text">My Recent Works</h3>
+      <div class="recent-works"><h2 class="hey-text">My Recent Works</h2>
```
Comment (mercymugambi) [inline index.html:61]: Update the headings to follow the standards for the hierachy structure

File: index.html [hunk 10]
```diff
@@ -68,7 +68,7 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
           <div class="card"></div>
         
           <div class="project-work">
-            <h4 class="title-1">Multi-Post Stories<br>Gain+Glory</h4>
+            <h3 class="title-1">Multi-Post Stories<br>Gain+Glory</h3>
```
Comment (mercymugambi) [inline index.html:71]: Update the headings to follow the standards for the hierachy structure

File: index.html [hunk 11]
```diff
@@ -85,7 +85,7 @@ <h4 class="title-1">Multi-Post Stories<br>Gain+Glory</h4>
...
```
Comment (mercymugambi) [inline index.html:88]: Update the headings to follow the standards for the hierachy structure

File: style.css [hunk 12]
```diff
@@ -980,6 +988,11 @@ h4 {
     width: 10%;
     margin-top: 20%;
     margin-left: 45%;
+    transition: all 0.5s ease-out;
```
Comment (mercymugambi) [inline style.css:991]: Add transition and animation to icon to make them interactive

File: style.css [hunk 13]
```diff
@@ -980,6 +988,11 @@ h4 {
...
+  }
+
+  .icons:hover {
+    transform: scale(1.04);
```
Comment (mercymugambi) [inline style.css:995]: Add transition and animation to icon to make them interactive

File: style.css [hunk 14]
```diff
@@ -992,5 +1005,10 @@ h4 {
     margin-top: 5%;
     margin-left: 45%;
     margin-bottom: 10%;
+    transition: all 0.5s ease-out;
```
Comment (mercymugambi) [inline style.css:1008]: Add transition and animation to icon to make them interactive

File: style.css [hunk 15]
```diff
@@ -992,5 +1005,10 @@ h4 {
...
+  }
+
+  .iconns:hover {
+    transform: scale(1.04);
```
Comment (mercymugambi) [inline style.css:1012]: Add transition and animation to icon to make them interactive

**Review by wayungi — COMMENTED** [review]
Hi @valt19 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [x] Please remove all lines starting with `>` in your README file. These lines offer you guidance on how to fill in the file which should be deleted afterward. 🎟️ 
- [x] You have also incorrectly filled in most of the fields in your README. Kindly avoid copying templates from other projects and paste them without editing. 🎫 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by valt19** [comment]
As requested by the previous code reviewer, changes have been implemented.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @valt19 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**